### DISCUSSION TITLE
Feature: Posibility to disable access log

### DISCUSSION THREAD

**Discussion by Tjerk-Haaye-Henricus** [docker-library/wordpress#778] [General]
To improve Debugging it would be so nice to have the possibility to disable the access log. For one request i see over 100 rows of access logs in dev and i have to search for my debug logs all the time, which are much more important to me.

**Comment by tianon** [comment]
This one probably requires modifying the Apache configuration, so isn't something we'll add to the image itself. :see_no_evil:

You _might_ be able to do something clever with `-c` or `-C` passed to `apache2-foreground`, but I don't know that you'd be able to scope it correctly for it to work the way you want.
